packages/new-polymath-ui/src/components/ModalTransactionQueue/TransactionItem/TransactionItem.tsx

```diff
@@ -24,27 +23,31 @@ interface TransactionItemProps {
 }
 
 const getIcon = (transaction: types.TransactionEntity) => {
```

Comment:
Repeated use of `transaction.status` warrants defining `const { status } = transaction;` and using `status` accross the function.